NobankCore/noBank/src/main/java/com/hunk/nobank/activity/dashboard/DashboardPresenter.java

```diff
@@ -1,47 +1,41 @@
-package com.hunk.nobank.activity;
+package com.hunk.nobank.activity.dashboard;
 
-import android.app.Activity;
-import android.os.Bundle;
-import android.support.annotation.Nullable;
-import android.support.v4.app.Fragment;
-import android.view.LayoutInflater;
-import android.view.View;
-import android.view.ViewGroup;
-import android.widget.ImageView;
-import android.widget.TextView;
+import android.support.annotation.VisibleForTesting;
 
 import com.hunk.nobank.Core;
-import com.hunk.nobank.R;
 import com.hunk.nobank.contract.AccountType;
 import com.hunk.nobank.manager.AccountDataManager;
-import com.hunk.nobank.manager.dataBasic.ManagerListener;
 import com.hunk.nobank.manager.UserManager;
 import com.hunk.nobank.manager.dataBasic.ViewManagerListener;
 import com.hunk.nobank.model.AccountSummaryPackage;
 
-public class BalanceFragment extends Fragment {
+/**
+ * @author HunkDeng
+ * @since 2016/5/23
+ */
+public class DashboardPresenter {
+    @VisibleForTesting
+    private DashboardView mView;
 
-    private TextView mBalance;
-    private ImageView mUserLogo;
     private UserManager mUserManager;
     private AccountDataManager mMainAccountDataManager;
     private AccountDataManager mVaultAccountDataManager;
+
+    @VisibleForTesting
     private ViewManagerListener mViewManagerListener = new ViewManagerListener(this) {
         @Override
         public void onSuccess(String managerId, String messageId, Object data) {
-            if (managerId.equals(mUserManager.getManagerId())) {
```

Comment:
use UserManager.MANAGER_ID will be better than using mUserManager.getManagerId().